docs/guide/src/example-plugin/cmd.go

```diff
@@ -29,8 +36,17 @@ var (
 	}
 )
 
+//Send a transaction
 func cmdExamplePluginTx(c *cli.Context) error {
+	//Retrieve any flag results
 	exampleFlag := c.Bool("valid")
+
+	//Create a transaction object with flag results
 	exampleTx := ExamplePluginTx{exampleFlag}
-	return commands.AppTx(c, "example-plugin", wire.BinaryBytes(exampleTx))
+
+	//Encode transaction bytes
+	exampleTxBytes := wire.BinaryBytes(exampleTx)
+
+	//Send the transaction and return any errors
```

Comment:
maybe add another comment about how the exampleTx is serialized and used as the `data` in the AppTx